core/jobs/src/main/java/com/cognifide/aet/job/common/modifiers/click/ClickModifier.java

```diff
@@ -50,19 +50,12 @@ public CollectorStepResult collect() throws ProcessingException {
           .waitForElementToBeClickable(webDriver, getLocator(), getTimeoutInSeconds());
       WebElement elementToClick = webDriver.findElement(getLocator());
 
-      if (elementToClick.isDisplayed()) {
-        elementToClick.click();
-        result = CollectorStepResult.newModifierResult();
-      } else {
-        final String message = String
-            .format("Element defined by %s is not yet visible!", getLocator().toString());
-        result = CollectorStepResult.newProcessingErrorResult(message);
-        LOG.warn(message);
-      }
+      elementToClick.click();
+      result = CollectorStepResult.newModifierResult();
     } catch (WebDriverException e) {
       final String message =
-          String.format("No element defined by %s could be found before timeout (%s seconds)!",
-              getLocator().toString(), getTimeoutInSeconds());
+          String.format("Element not found before timeout (%s seconds): %s!",
+                  getTimeoutInSeconds(), getLocator().toString());
       LOG.warn(message, e.getMessage());
```

Comment:
Why log the message and not the throwable itself? Are we hiding the stack trace on purpose?